File: FrontEnd/DevKiller/app/src/main/java/com/ryansatj/devkill_android/model/Project.java
package com.ryansatj.devkill_android.model;

public class Project {
    public int id;
    public String repository;
    public int userid;
    public String title;
    public String descriptions;

    public Project(int id, String repository, int userid, String title, String descriptions){
        this.repository =repository;
        this.id = id;
        this.userid = userid;
        this.title = title;
        this.descriptions = descriptions;
    }
    public String toString() {
        return "Project{id=" + id + "repository=" + repository + ", userid=" + userid + ", title='" + title + "', descriptions='" + descriptions + "'}";
    }
}

File: Backend/_models/project.js
class Project{
    constructor(id, repository, userid, title, description){
        this.id = id
        this.repository = repository
        this.userid = userid
        this.title = title
        this.description = description
    }

    addSection(section){
        this.sections.push(section);
    }
}

module.exports = Project;